settings.py

```diff
@@ -62,14 +62,24 @@ def celery_queue(name):
 #: celery broker
 BROKER_URL = env('CELERY_BROKER_URL', RABBIT_MQ_URL)
 CELERY_BROKER_URL = BROKER_URL
+CELERY_TASK_ALWAYS_EAGER = strtobool(env('CELERY_TASK_ALWAYS_EAGER', 'False'))
 #: number retry when task fail
-NUMBER_RETRY = int(env('NUMBER_RETRY', 3))
-BROKER_CONNECTION_MAX_RETRIES = NUMBER_RETRY
+MAX_RETRIES = int(env('MAX_RETRIES', 3))
+BROKER_CONNECTION_MAX_RETRIES = MAX_RETRIES
 
-#: allow agent
-AGENT_ALLOW = json.loads(env('AGENT_ALLOW', '["superdesk", "python-requests" ,"postmanruntime", "mozilla"]'))
 #: Codec support
-CODEC_SUPPORT = json.loads(env('CODEC_SUPPORT', '["vp8", "vp9", "h264", "theora", "av1"]'))
+CODEC_SUPPORT_VIDEO = json.loads(env('CODEC_SUPPORT_VIDEO', '["vp8", "vp9", "h264", "theora", "av1"]'))
```

Comment:
imo there is no need to have such things in env, we can just use list here (and below)